macos/JackTrip.app_template/Contents/Info.plist

```diff
@@ -11,15 +11,15 @@
 	<key>CFBundleIconFile</key>
 	<string>jacktrip</string>
 	<key>CFBundleIdentifier</key>
-	<string>org.psi-borg.qjacktrip</string>
+	<string>%BUNDLEID%</string>
 	<key>CFBundleInfoDictionaryVersion</key>
 	<string>6.0</string>
 	<key>CFBundleName</key>
-	<string>QJackTrip</string>
+	<string>%BUNDLENAME%</string>
```

Comment:
Does the `%BUNDLENAME%` ever get set? I might be missing something, but I can't find it.

Reply:
Yep, in the assemble_app script (line 24):
`sed -i '' "s/%BUNDLENAME%/$APPNAME/" "$APPNAME.app/Contents/Info.plist"`

Reply:
Ah thanks, sorry I missed it!